aospy/_constants.py

```diff
@@ -0,0 +1,14 @@
+"""commonly used physical constants"""
```

Comment:
nit: let's say "Physical constants used in aospy", because this is by no means an exhaustive list of commonly used physical constants :)